src/com/core/tictactoe/App.java

```diff
@@ -0,0 +1,10 @@
+package com.core.tictactoe;
+
+public class App {
+
+    public static void main(String[] args) {
+        CommandLineUI commandLineUI = new CommandLineUI(System.out, System.in);
+        GamesController gamesController  = new GamesController(commandLineUI);
+        gamesController.gamesMenu();
```

Comment:
I feel like this should be named simply 'run()' as it is doing more than just games menu functionality.

Reply:
Done.